README.adoc

```diff
@@ -109,27 +109,49 @@ Navigate to `{name-api}` and use the username `bob` and the password `bobpwd` to
 
 Similarly, navigate to `{ping-api}/name-service` and you will see `pong`.
 
-== Modify name application
+== Modifying name microservice
 
-First, update `name` to have a configurable greeting message. At the moment, it is hardcoded to be `Hello!`. Use MicroProfile Config and CDI to inject the value of an environment variable called `GREETING` into the `greeting` member of the `NameResource` class. Then use `greeting` to create a response with a custom greeting message.
+Update `name` to have a configurable greeting message. At the moment, it is hardcoded to be `Hello!`. Use MicroProfile Config and CDI to inject the value of an environment variable called `GREETING` into the `greeting` member of the `NameResource` class. Then use `greeting` to create a response with a custom greeting message.
 
-Update the name resource class in the `name/src/main/java/io/openliberty/guides/name/NameResource.java` file:
+Update the name resource class in the `name/src/main/java/io/openliberty/guides/name/NameResource.java` file.
+
+First, inject the greeting configuration into the resource:
+[source, java, indent=0]
+----
+include::finish/name/src/main/java/io/openliberty/guides/name/NameResource.java[tags=greeting]
+----
+
+Next, update the response to use the configured greeting:
```

Comment:
perhaps to mention "in the `getContainerName` method, ie, "Next, update the response to use the configured greeting in the `getContainerName`:" to be more specific.